README.md

```diff
@@ -0,0 +1,69 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MATH-MAGICIANS
+
```

Comment:
- [ ] You have duplicate files in you project. you can remove some.
![Screenshot from 2022-04-26 12-38-50](https://user-images.githubusercontent.com/65860678/165282413-9154f109-86d9-4fce-a9f1-76927e111577.png)


- [ ] node_modules should not be pushed to your github. You can check this [link](https://git-scm.com/docs/gitignore) to know why and how to do it.